src/main/scala/com/codacy/client/bitbucket/v2/service/PullRequestServices.scala

```diff
@@ -27,19 +31,26 @@ class PullRequestServices(client: BitbucketClient) {
    * Gets the list of commits of a pull request
    *
    */
-  def getPullRequestCommits(owner: String, repository: String, prId: Long): RequestResponse[Seq[SimpleCommit]] = {
-    val url = s"https://bitbucket.org/api/2.0/repositories/$owner/$repository/pullrequests/$prId/commits?pagelen=100"
+  def getPullRequestCommits(
```

Comment:
Codacy found an issue: [Do not prefix getters with 'get'](https://app.codacy.com/gh/codacy/bitbucket-scala-client/pullRequest?prid=5907044)